client/table.go

```diff
@@ -37,7 +37,7 @@ type column struct {
 	width int
 }
 
-var headerSlice = make([]header, 16)
+var headerSlice = make([]header, 23)
```

Comment:
```suggestion
var headerSlice = make([]header, HumanTime+1)
```
(or add a unexported `headerLen` to the iota)